config/vufind/ProQuestFSG.ini

```diff
@@ -0,0 +1,160 @@
+; PrimoCentral without checking the local availability against a holdingsfile.
+[CheckboxFacets]
+limitFullText = "fulltext_limit"
+limitScholarlyJournals = "scholarly_limit"
+limitPeerReviewed = "peer_reviewed_limit"
+
+; Facet display settings
+[Results_Settings]
+; By default, the side facets will only show 6 facets and then the "show more"
+; button. This can get configured with the showMore settings.
+; You can use the * to set a new default setting.
+showMore[*] = 6
+; Or you can set a facet specific value by using the facet name as index.
+;showMore['rtype'] = 10
+; Do we want any facets to be collapsed by default?
+;collapsedFacets = *
+
+; This section shows which search types will display in the basic search box at
+; the top of Primo pages.  The name of each setting below corresponds with an
+; index defined in the Primo API.  The value of each setting is the text to
+; display on screen.  All on-screen text will be run through the translator, so
+; be sure to update language files if necessary.  The order of these settings
+; will be maintained in the drop-down list in the UI.
+[Basic_Searches]
+cql.serverChoice    = "All Fields"
+title               = Title
+author              = Author
+subject             = Subject
+publication         = Journal Title
+;abstract            = Abstract
+
+; This section defines which search options will be included on the advanced
+; search screen.  All the notes above [Basic_Searches] also apply here.
+[Advanced_Searches]
+cql.serverChoice    = "All Fields"
+title               = Title
+author              = Author
+subject             = Subject
+publication         = Journal Title
+;abstract            = Abstract
+
+; This section defines the sort options available on search results.
+[Sorting]
+relevance = sort_relevance
+date/descending = sort_year
+date/ascending = sort_year_asc
+
+; This section allows you to specify hidden sorting options. They can be used to create a
+
+; This section controls the behavior of the ProQuestFSGRecord module.
+[Record]
+; Set this to true in order to enable "next" and "previous" links to navigate
+; through the current result set from within the record view.
+next_prev_navigation = false
+
+; This section controls what happens when a record title in a search result list
+; is clicked. VuFind can either embed the full result directly in the list using
+; AJAX or can display it at its own separate URL as a full HTML page.
+; full - separate page (default)
+; tabs - embedded using tabs (see record/tabs.phtml)
+; accordion - embedded using an accordion (see record/accordion.phtml)
+; NOTE: To turn this feature on for favorite lists, see the lists_view setting
+; in the [Social] section of config.ini.
+; NOTE: This feature is incompatible with SyndeticsPlus content; please use
+;       regular Syndetics if necessary.
+[List]
+view=full
+
+; This section provides settings for optional caching of search requests.
+[SearchCache]
```

Comment:
I'm pretty sure that your connector here does not support search caching. Am I missing something, or should we remove these settings for now?

Reply:
This should be working!  It's actually the SRU Connector that uses ConnectorCacheTrait and uses it in search(); looks like I added that in this PR.

Reply:
Fantastic! Sorry for missing that detail -- didn't look high enough in the class hierarchy.